### ISSUE TITLE
[Accessibility]A11y_ Java Tooling for IntelliJ/Eclipse_Select JSP Template: The information that a template is selected is not conveyed properly to the user

### ISSUE THREAD

**Issue by Tanushagottipati** [microsoft/azure-tools-for-java#3333]
[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faccessibilityinsights.io%2F&data=02%7C01%7Cv-tagott%40microsoft.com%7Cb67b2c4b646d4f9561a208d6f4b5c39b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636965458847261333&sdata=PJMiKjb64uwhBN98LZpctnBiAVeOvw%2BLBrnxBtrjl2M%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_Java Tooling for IntelliJ/Eclipse-Stay Healthy; #MAS1.3.1; #C+E-H; #Java tooling for IntelliJ/Eclipse

**Environment Details:**
Eclipse IDE for Enterprise Java Developers.
Version: 2019-06 (4.12.0)
Build id: 20190614-1200
Solution Installed: The Azure Toolkit for Eclipse 3.17.0

**OS Details:** 
OS: Windows (Version 1903, Build: 18362.175)
MAS Violated: MAS1.3.1

**Note: Similar issue is found for ‘Deploy a WebApp’ popup, ‘Create New Web App’ popup**

**Repro Steps:** 
1) Open Java EE - Eclipse application
2) Install 'Azure Toolkit for Eclipse' from Marketplace settings/preferences popup
3) Tab till tools-->Azure-->Azure Sign in and sign into Azure account and select any subscription
4) Tab till File-->New-->Project--->Dynamic Web Project
5) Within Eclipse's Project Explorer view, expand Newly created dynamic web project. Right-click WebContent, click New-->JSP File
6) Tab till Next button and press enter
7) Select any template present in the table
8) Verify if the user can know difference between selected and Non-selected templates

**Actual Result:** 
The information that a template is selected is not conveyed properly to the user. There is no difference between selected and non-selected templates

**Expected Result:** 
There should be difference between selected and non- selected templates. There should be a proper color difference between selected and Non-selected templates 

**User Impact:** 
Users will not know the difference between selected and non-selected template

**MAS Reference:** 
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

![Bug_12_Create New Web App](https://user-images.githubusercontent.com/41109135/62205097-467f3780-b3ac-11e9-9d0c-0cac41c8d6b7.png)
![Bug_12_Deploy Webapp](https://user-images.githubusercontent.com/41109135/62205099-4717ce00-b3ac-11e9-94f1-72418aeae2b5.png)
![Bug_12_Selected list item cannot be seen properly by the user](https://user-images.githubusercontent.com/41109135/62205100-4717ce00-b3ac-11e9-923c-a0958b1ce4f2.png)

**Comment by Flanker32** [comment]
The behavior(border, back ground color) of selected item is controlled by Eclipse theme, which is out of the scope of the toolkit,besides, eclipse provide multi theme and this issue will not appear in other themes like Light. Here are the screen shots:

![image](https://user-images.githubusercontent.com/12445236/64336638-10b71980-d010-11e9-9cc6-19b8dd888b30.png)

![image](https://user-images.githubusercontent.com/12445236/64336830-8622ea00-d010-11e9-86a8-e5a10c20229d.png)

![image](https://user-images.githubusercontent.com/12445236/64336888-a783d600-d010-11e9-9f9d-a35b534ba201.png)

**Comment by Flanker32** [comment]
Importance of experience: Medium, users will not know the difference between selected and non-selected items in Eclipse table.
Quality of experience: not impacted. Eclipse provide alter theme for users, for instance, light, which has more obvious difference between selected and non-selected items
Alternate solutions: No alternate technologies or options can be leveraged.
Evidence: All evidence is provided in this issue and external issue.

**Comment by Flanker32** [comment]
@Tanushagottipati Please help us review this issue, thanks!

**Comment by Tanushagottipati** [comment]
External Bug: https://bugs.eclipse.org/bugs/show_bug.cgi?id=550837

**Comment by Flanker32** [comment]
Discussed this bug with Ryan and Sarah and approved to close this issue as its external and there is non blocking scenario.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749784
